### PR TITLE
Web metadata change

### LINKED ISSUE #44: [Misc] Modify metadata
Modify the metadata "Web site created using create-react-app"
Screenshot 2023-11-28 at 5 51 12 PM

### PR DESCRIPTION
**Pull request by whuang602**
**Merge #48 first**

- changed the website's metadata from default "web site created with create-react-app" to "UCSB CS156 Project Organic"
This can be tested whenever you paste the link somewhere (especially on slack), it will now give the viewer an accurate description of what the app actually is.
Screenshot 2023-11-28 223253

close #44

### CODE REVIEW

**Review by justinchung712 — APPROVED** [review]
lgtm

**Review by pconrad — CHANGES_REQUESTED** [review]
Some feedback to consider...

File: frontend/public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Organic</title>
+    <title>CS156 Project Organic</title>
```
[the remark this answers is not in the document]
Comment (whuang602) [reply frontend/public/index.html]: Understood, changed it back!

**Review by pconrad — APPROVED** [review]
LGTM

**Review by SparklyJeff — APPROVED** [review]
LGTM

**Review by vedantshah99 — APPROVED** [review]
lgtm

**Pull request merged**